### PR TITLE
Added support for antenna creation and map antenna markers

### LINKED ISSUE #148: Support for RFID devices
As a developer I'd like to have support for RFID devices

### PR DESCRIPTION
**Pull request by jarinykanen**
Resolves #148

### CODE REVIEW

File: src/components/screens/floor-plan-screen.tsx [hunk 1]
```diff
@@ -891,10 +919,91 @@ export class FloorPlanScreen extends React.Component<Props, State> {
     );
   }
 
+  /**
+   * Event handler for add device click
+   *
+   * @param deviceToCreate exhibition device to create
+   */
```
Comment (lauriLukkarinen) [inline src/components/screens/floor-plan-screen.tsx:926]:
```suggestion
  /**
   * Event handler for add antenna click
   *
   * @param antennaToCreate exhibition antenna to create
   */
```

File: src/components/generic/floor-plan-map.tsx [hunk 2]
```diff
@@ -530,6 +622,46 @@ export default class FloorPlanMap extends React.Component<Props, State> {
...
+    }
+
+    if (selectedItems.antenna) {
+      tempMapData.antennas = [selectedItems.antenna];
+    }
+
+    const tempLeafletIdToAntennaMap = new Map<number, RfidAntenna>();
+
+    tempMapData.antennas.forEach(antenna => {
+
```
Comment (anttileppa) [inline src/components/generic/floor-plan-map.tsx:643]:
```suggestion
```

File: src/components/floor-plan/floor-plan-info.tsx [hunk 3]
```diff
@@ -180,6 +217,21 @@ class FloorPlanInfo extends React.Component<Props, State> {
...
   }
+
+  /**
+   * Get device group select items
+   */
+  private getDeviceGroupItems = () => {
+    const { deviceGroups } = this.props;
+
+    if (!deviceGroups) {
+      return (<div/>);
```
Comment (anttileppa) [inline src/components/floor-plan/floor-plan-info.tsx:228]: why empty div instead of null?

File: src/components/floor-plan/floor-plan-info.tsx [hunk 4]
```diff
@@ -180,6 +217,21 @@ class FloorPlanInfo extends React.Component<Props, State> {
...
+    }
+    const items = deviceGroups.map(group => {
```
Comment (anttileppa) [inline src/components/floor-plan/floor-plan-info.tsx:230]:
```suggestion

    return deviceGroups.map(group => {
```

File: src/components/floor-plan/floor-plan-info.tsx [hunk 5]
```diff
@@ -180,6 +217,21 @@ class FloorPlanInfo extends React.Component<Props, State> {
...
+      return <MenuItem key={ group.id } value={ group.id }>{ group.name }</MenuItem>
+    });
+    return items;
```
Comment (anttileppa) [inline src/components/floor-plan/floor-plan-info.tsx:233]:
```suggestion
```

File: src/components/screens/floor-plan-screen.tsx [hunk 6]
```diff
@@ -891,10 +919,91 @@ export class FloorPlanScreen extends React.Component<Props, State> {
...
+   * Event handler for delete antenna click
+   */
+  private onAntennaDeleteClick = async () => {
+    const { accessToken, exhibitionId } = this.props;
+    const { selectedAntenna } = this.state;
+    if (!exhibitionId || !selectedAntenna || !selectedAntenna.id) {
+      return;
+    }
+
+    // FIXME: Add nicer dialog
```
Comment (anttileppa) [inline src/components/screens/floor-plan-screen.tsx:951]: Should this FIXME be fixed in this PR?

Comment (jarinykanen) [reply src/components/screens/floor-plan-screen.tsx:951]: No, this is done in seperate task

**Pull request merged**